_posts/2017-08-08-CompetitiveCoding.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Competitive Coding: From Novice to Expert"
```

Comment:
Wow!! ref: stages of skill acquisition .........the title would trouble you so much....anyways changed it..